bundles/org.openhab.binding.teslapowerwall/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.teslapowerwall</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: TeslaPowerwall Binding</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>javax.measure</groupId>
```

Comment:
javax.measure is used in many binding, but none of them has this dependency. I guess this is not needed.

Reply:
It builds without it,  so removed :)